CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+Please start by discussing the change via [issue](https://github.com/LDSSA/curriculum-development/issues).
```

Comment:
Are you expecting people to open an issue?

Reply:
Yes, the proposal is to discuss and troubleshoot ideas there first, before implementing them. What do you think?

Reply:
At first, I considered two alternatives: the `#curriculum` Slack cannel and raising issues.

I think things get lost on Slack most times.

Alternatively, one could do the pull request right away, but an issue is less expensive in terms of effort and lower commitment.

Reply:
Also, it's important to discuss the proposed changes in some detail before implementation.

Reply:
Sure thing makes sense. Maybe be explicit about opening a new issue with a good subject line.

Reply:
And use appropriate tags as well.